templates/my.cnf.erb

```diff
@@ -0,0 +1,9 @@
+[client]
```

Comment:
If we are okay with adding a dependency, I think that it would be better to use the [inifile](https://forge.puppet.com/puppetlabs/inifile) module here.